contribution-guidelines.md

```diff
@@ -32,6 +32,8 @@ Please take a moment to check whether an issue already exists. If it does, it ne
 
 We are always thrilled to receive pull requests, and do our best to process them as quickly as possible. Not sure if that typo is worth a pull request? Do it! We will appreciate it.
 
+If you don't get some response to a pull request within a day, please do mention it in the #ipfs channel, in case it has gotten forgotten.
```

Comment:
Yeah, "within a few days" feels more appropriate than "within a day" for
sure. Not sure it's realistic to be able to review any PR within a day...

Sincerely,
Victor Bjelkholm

On Mon, Sep 5, 2016 at 5:48 PM, Richard Littauer <notifications@github.com>
wrote:

> In contribution-guidelines.md
> <https://github.com/ipfs/community/pull/165#discussion_r77538775>:
>
> > @@ -32,6 +32,8 @@ Please take a moment to check whether an issue already exists. If it does, it ne
> >
> >  We are always thrilled to receive pull requests, and do our best to process them as quickly as possible. Not sure if that typo is worth a pull request? Do it! We will appreciate it.
> >
> > +If you don't get some response to a pull request within a day, please do mention it in the #ipfs channel, in case it has gotten forgotten.
>
> "If you don't get a response to a pull request within a few days, feel
> free to mention it in the #ipfs IRC channel. We may be able to get to it
> faster." Sound better?
>
> Doesn't put the pressure of a day on us. It also probably wasn't
> forgotten; we've got a lot of priorities.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ipfs/community/pull/165/files/3a12efa69f9a4622acce8fbcd57454ae7381f444#r77538775>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAcD9O4rIe8aR1VfG_PriJyGy8gJvWZsks5qnDnjgaJpZM4J0clC>
> .
>